### ISSUE TITLE
There is no instructions for screen reader user use "ctrl+K and ctrl+I" to select suggestions: A11y_ Visual Studio Code Client_ Inline Suggestions _ View the tooltip for the suggestion and interact with it _Labels or instructions.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#250460]
Please do not close this bug. This bug should only be closed by Trusted Tester after verification.
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_Visual Studio Code Client Copilot_Win32_June2025; #Visual Studio Code Client; #WCAG3.3.2; #FTP; #Labels or Instructions; #AILimited; #NVDA; #Win11; #VSCode2025;

### Actual Result:
There is no instructions for screen reader user use "ctrl+K and ctrl+I" to select suggestions.
### Expected Results:
Instructions should be provided for suggestions popup like use “ctrl+K and ctrl+I” to select the suggestions.
### Environment Details:
Application Name: Visual Studio Code Client Win32
Visual Studio Code insider Version: 1.101.0 – insider(user setup)
Edition Windows 11 Enterprise
Version 24H2 OS build 26100.4202
NVDA version: 2024.4.2(2024.4.2.35031), JAWS.
### Repro Steps:

1. Open the "Visual Studio Code" dialog box.
2. In Visual Studio Code, create a new JavaScript (*.js) file.
3. In the code Paste this code function calculateDaysBetweenDates(begin, end) { and press enter.
4. A suggestion will appear.
5. Press "Ctrl + K, Ctrl + I" and a tooltip will appear.
6. Observe that There is no instructions for screen reader user use "ctrl+K and ctrl+I" to select suggestions.

### User Impact:
Screen Reader users will face difficulty of accessing the suggestions without instructions Provided for those.
### Attachment:

https://github.com/user-attachments/assets/0252d5bd-c4f5-40f3-990b-14793286032e

**Comment by meganrogge** [comment]
we can add a hint to the alert

**Comment by meganrogge** [comment]
`ctrl+k+ctrl+i` is a well known thing for keyboard / screen reader users. However, I think the case you're describing is for inline suggestions, where a user can navigate between suggestion possibilities after pressing `ctrl+k+ctrl+i`, so that's what i'll fix.

<img width="419" alt="Image" src="https://github.com/user-attachments/assets/0e165833-0448-4901-b17f-388fc40cf2e3" />

**Comment by meganrogge** [comment]
I had forgotten that the reason we don't do something like that is we indicate in the aria label that users should open the accessible view, where they have access to all of those actions. This is working as designed. 

<img width="792" alt="Image" src="https://github.com/user-attachments/assets/5cd30893-13db-41b0-855f-651206365fb8" />

**Comment by keerthiduvvuri** [comment]
The shortcut key is documented [https://code.visualstudio.com/shortcuts/[keyboard-shortcuts-windows.pdf](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf)](https://code.visualstudio.com/shortcuts/%5Bkeyboard-shortcuts-windows.pdf%5D(https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf)). Hence closing the bug.
